### PR TITLE
Explain target property in errors, link to UI5 OData model

### PR DESCRIPTION
**Pull request by David-Kunz**

### CODE REVIEW

**Review by hm23 — APPROVED** [review]

File: node.js/events.md [hunk 1]
```diff
@@ -421,6 +421,11 @@ if (req.errors) //> get out somehow...
 - `target` _String (Optional)_ - The name of an input field/element a message is related to.
 - `args` _Array (Optional)_ - Array of placeholder values. See [Localized Messages](cds-i18n) for details.
 
+::: warning :::
+The `target` property is evaluated by the UI5 OData model and needs to be set according to the [documentation](https://ui5.sap.com/#/topic/fbe1cb5613cf4a40a841750bf813238e).
```
Comment (renejeglinsky) [inline node.js/events.md:425]:
```suggestion
The `target` property is evaluated by the UI5 OData model and needs to be set according to [Server Messages in the OData V4 Model](https://ui5.sap.com/#/topic/fbe1cb5613cf4a40a841750bf813238e).
```

**Review by renejeglinsky — APPROVED** [review]

File: node.js/events.md [hunk 2]
```diff
@@ -421,6 +421,11 @@ if (req.errors) //> get out somehow...
 - `target` _String (Optional)_ - The name of an input field/element a message is related to.
 - `args` _Array (Optional)_ - Array of placeholder values. See [Localized Messages](cds-i18n) for details.
 
+::: warning :::
```
Comment (renejeglinsky) [inline node.js/events.md:424]:
```suggestion
::: warning  `target` property for UI5 OData model
```

File: node.js/events.md [hunk 3]
```diff
@@ -421,6 +421,11 @@ if (req.errors) //> get out somehow...
 - `target` _String (Optional)_ - The name of an input field/element a message is related to.
 - `args` _Array (Optional)_ - Array of placeholder values. See [Localized Messages](cds-i18n) for details.
 
+::: warning  `target` property for UI5 OData model
```
Comment (renejeglinsky) [inline node.js/events.md:424]: Is it a warning or rather a tip? From the linked docs I don't get the impression that sth breaks and also in our docs it'S marked as optional.

```suggestion
::: tip `target` property for UI5 OData model
```

Comment (renejeglinsky) [reply node.js/events.md:424]: @David-Kunz Would you agree that this is a tip?

**Pull request merged**